### ISSUE TITLE
Allow regular users to access custom mounts.

### ISSUE THREAD

**Issue by carlba** [meefik/linuxdeploy#182]
I have been banging my head hard against this issue now so I'll just ask.

I have created a custom mount to my external sdcard and it is working great I can access it using my nonroot user from within the chroot. I would however like to have write access to the mount but I can't get it to work. I have tried added my user to the aid_sdcard_rw group but it has no affect at all.

The mount is pointing to /storage/sdcard1

Is this a bug or I am just not understanding how to accomplish this :)

**Comment by rschuetzler** [comment]
Is this the same thing I reported [here](https://github.com/meefik/linuxdeploy/issues/150)?

**Comment by carlba** [comment]
Yes it seems to be the same issue I should have search better before posting.
